tests/publiccloud/run_ltp.pm

```diff
@@ -59,8 +59,10 @@ sub run {
     $instance->run_ssh_command(cmd => 'sudo zypper --no-gpg-checks --gpg-auto-import-keys -q in -y ' . $remote_rpm_path, timeout => 600);
     $instance->run_ssh_command(cmd => 'sudo CREATE_ENTRIES=1 /opt/ltp/IDcheck.sh', timeout => 300);
 
-    my $reset_cmd = '~/restart_instance.sh ' . get_required_var('PUBLIC_CLOUD_PROVIDER') . ' ';
+    my $provider_type = get_required_var('PUBLIC_CLOUD_PROVIDER');
+    my $reset_cmd     = '~/restart_instance.sh ' . $provider_type . ' ';
     $reset_cmd .= $instance->instance_id . ' ' . $instance->public_ip;
+    $reset_cmd .= ' ' . $provider->region if $provider_type eq 'GCE';
```

Comment:
maybe not necessary. You can keep 1 line here:

    my $reset_cmd = '~/restart_instance.sh ' . get_required_var('PUBLIC_CLOUD_PROVIDER') . ' ' . $provider->region;

the bash script will anyways ignore $ZONE for AWS and Azure.

Reply:
I think this way it is cleaner , if you not insist I would like to keep it like this :)